rules/opentitan.bzl

```diff
@@ -714,27 +719,30 @@ def opentitan_rom_binary(
             extract_sw_logs_db = device == "sim_dv",
             **kwargs
         ))
-        elf_name = "{}.{}".format(devname, "elf")
-        bin_name = "{}_{}".format(devname, "bin")
 
-        # Generate Un-scrambled ROM VMEM
-        vmem_name = "{}_vmem".format(devname)
-        dev_targets.append(":" + vmem_name)
-        bin_to_vmem(
-            name = vmem_name,
-            bin = bin_name,
-            platform = platform,
-            word_size = 32,
-        )
+        # We only need to generate VMEM files for sim devices.
```

Comment:
It turns out that we use the vmem ROM files for splicing.

Reply:
ah yes, good catch. fixed.